install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-MBGLVERSION='3.4.1'
+MBGLVERSION='3.5.0'
```

Comment:
This won’t work until v3.5.0 is released. Let’s remove anything related to `install.sh` from this PR, as it shouldn’t be necessary.